### ISSUE TITLE
zhmc CLI requests password for every command in REPL mode.

### ISSUE THREAD

**Issue by leopoldjuergen** [zhmcclient/python-zhmcclient#176]
### Actual behavior
```
(demo) vagrant@vagrant-ubuntu-trusty-64:~$ zhmc -h ZHMC_IP -u ZHMC_USER
> cpc list
-Enter password (for user pedebug at HMC ZHMC_IP): 
|+----------+------------------+
| name     | status           |
|----------+------------------|
| P000TEST | service-required |
+----------+------------------+
> cpc list
Enter password (for user pedebug at HMC ZHMC_IP):
```

### Expected behavior
Password should be provided once.

### Execution environment

* zhmcclient version: 0.9.0
* Operating system (type+version): Ubuntu 14.04

**Comment by andy-maier** [comment]
Fixed in PR #227.

**Comment by andy-maier** [comment]
This issue has been copied to zhmcclient/zhmccli#8
